<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="dayshours">
      <attribute name="title">
       <string>Days &amp; Hours</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Number of Days</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Number of Hours</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="nDays">
         <property name="toolTip">
          <string>Number of days in the school week</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="nHours">
         <property name="toolTip">
          <string>Number of hours each day (max, then they can be blocked in fet)</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QListWidget" name="dayNames">
         <property name="toolTip">
          <string>Name of the days of the week</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QListWidget" name="hourNames">
         <property name="toolTip">
          <string>Name or text to be displayed for each hour</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="addDay">
         <property name="toolTip">
          <string>Add another day to the list</string>
         </property>
         <property name="text">
          <string>Add Day</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="addHour">
         <property name="toolTip">
          <string>Add another hour to the list</string>
         </property>
         <property name="text">
          <string>Add Hour</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="removeDay">
         <property name="toolTip">
          <string>Remove the selected day from the list</string>
         </property>
         <property name="text">
          <string>Remove Day</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="removeHour">
         <property name="toolTip">
          <string>Remove the selected hour from the list</string>
         </property>
         <property name="text">
          <string>Remove Hour</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label</zorder>
      <zorder>label_3</zorder>
      <zorder>nDays</zorder>
      <zorder>nHours</zorder>
      <zorder>dayNames</zorder>
      <zorder>hourNames</zorder>
      <zorder>addDay</zorder>
      <zorder>removeDay</zorder>
      <zorder>addHour</zorder>
      <zorder>removeHour</zorder>
     </widget>
     <widget class="QWidget" name="various">
      <attribute name="title">
       <string>Various</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>School Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="schoolname">
         <property name="toolTip">
          <string>School name that will apears in fet file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>CSV's activities Identifiers</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Independent activity</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="independentActivity">
            <property name="toolTip">
             <string>Activities that are not linked with others</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Option activity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="optionActivity">
            <property name="toolTip">
             <string>Activities that students can choose among several of them</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Small Group Activity</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="smallgroupActivity">
            <property name="toolTip">
             <string>Activities that are tought with 3 teachers simultaneously in 2 groups making smaller groups</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Meeting activity</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="meetingActivity">
            <property name="toolTip">
             <string>Teachers meetings</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Guard Activity</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="guardActivity">
            <property name="toolTip">
             <string>Teachers to guard when others can't be in their classes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Guards</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Simultaneous Guard Teachers (max)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="nguards">
            <property name="toolTip">
             <string>How many teachers have to be on guard each hour. Max, then the rest can be deleted.</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
